client/src/store/state.js

```diff
@@ -0,0 +1,22 @@
+export const state = {
+  companyValues: [], // ARRAY OF COMPANY VALUES IN DATABASE
+  rankedUsers: [], // ARRAY OF USERS RANKED BY SHOUTOUTS GIVEN / RECEIVED
+  recentShoutouts: [], // ARRAY OF THE MOST RECENT SHOUTOUTS FROM DATABASE
+  shoutouts: [], // ARRAY OF QUERIED SHOUTOUTS RETRIEVED FROM DATABASE
+  users: [], // ARRAY OF ALL USERS IN DATABASE
```

Comment:
Are the comments necessary here? I feel like the prop names do a pretty good job of being self-explanatory, but if they are unclear it may make sense to choose a more descriptive prop name

Reply:
They may not be strictly necessary, no. I wanted to make sure that it was clear what was connected to what